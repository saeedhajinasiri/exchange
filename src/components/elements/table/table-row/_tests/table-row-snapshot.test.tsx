import React from "react";
import {cleanup, render} from "test";
import TableRow from '../table-row'
import Table from '../../table'
import TableHead from '../../table-head'
import TableCell from '../../table-cell'

afterEach(cleanup)

describe('Test TableRow UI', () => {
  it('Default', () => {
    const {asFragment} = render(<Table><TableHead><TableRow/></TableHead></Table>);
    expect(asFragment()).toMatchSnapshot();
  });

  it('Test with children prop', () => {
    const children = <TableCell>children</TableCell>
    const {asFragment} = render(<Table><TableHead><TableRow>{children}</TableRow></TableHead></Table>);
    expect(asFragment()).toMatchSnapshot();
  });
});
