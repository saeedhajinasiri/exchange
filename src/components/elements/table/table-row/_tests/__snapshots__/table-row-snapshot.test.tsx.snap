// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test TableRow UI Default 1`] = `
<DocumentFragment>
  <table
    class="w-full"
  >
    <thead
      class="table-header-group"
    >
      <tr
        class="border-b border-grey-100 border-opacity-20 hover:bg-opacity-75 bg-white border-grey-light"
      />
    </thead>
  </table>
</DocumentFragment>
`;

exports[`Test TableRow UI Test with children prop 1`] = `
<DocumentFragment>
  <table
    class="w-full"
  >
    <thead
      class="table-header-group"
    >
      <tr
        class="border-b border-grey-100 border-opacity-20 hover:bg-opacity-75 bg-white border-grey-light"
      >
        <td
          class="px-4 bg-opacity-20 min-w-[125px] h-10"
        >
          <p
            class="text-m-sm sm:text-t-sm md:text-d-sm text-grey text-left font-regular"
          >
            children
          </p>
        </td>
      </tr>
    </thead>
  </table>
</DocumentFragment>
`;
